crates/runtime/src/tools/catalog.rs

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2024 The Spice.ai OSS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+use async_trait::async_trait;
+
+use super::SpiceModelTool;
+use std::sync::Arc;
+
+#[async_trait]
```

Comment:
```suggestion
```

Doesn't seem to be used?

Reply:
Good point. I wanted async because future SpiceToolCatalog implementation may need to make network requests (e.g. to other spiced instances).